<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Araclar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(112, 112, 112);</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>222</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="7" column="5">
     <widget class="QPushButton" name="pushButton_sil1">
      <property name="text">
       <string>Sil</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="lineEdit_plaka"/>
    </item>
    <item row="5" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Marka</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_18">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_marka"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_aracTuru"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vites</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLineEdit" name="lineEdit_renk"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_yakit"/>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEdit_motorHacmi"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_vites"/>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_19">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Motor No</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_21">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Renk</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="lineEdit_kasa"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kasa Tipi</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QLineEdit" name="lineEdit_motor"/>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_23">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kapı</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_24">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Çekiş</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ARAC ID</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_20">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Üretim Yılı</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="pushButton_update1">
      <property name="text">
       <string>Güncelle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_yeni1">
      <property name="text">
       <string>Yeni</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLineEdit" name="lineEdit_cekis"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Motor Hacmi</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_25">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Araç Türü</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLineEdit" name="lineEdit_kapi"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_model"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_aracid">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_22">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Yakıt Türü</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_uretimYil"/>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Plaka</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="pushButton_kaydet1">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="pushButton_getir1">
      <property name="text">
       <string>Getir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>1201</width>
     <height>431</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>6</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>7</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>8</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>9</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>10</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>11</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>12</string>
    </property>
   </column>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_aracTuru</tabstop>
  <tabstop>lineEdit_kasa</tabstop>
  <tabstop>lineEdit_marka</tabstop>
  <tabstop>lineEdit_motorHacmi</tabstop>
  <tabstop>lineEdit_model</tabstop>
  <tabstop>lineEdit_cekis</tabstop>
  <tabstop>lineEdit_uretimYil</tabstop>
  <tabstop>lineEdit_kapi</tabstop>
  <tabstop>lineEdit_yakit</tabstop>
  <tabstop>lineEdit_renk</tabstop>
  <tabstop>lineEdit_vites</tabstop>
  <tabstop>lineEdit_motor</tabstop>
  <tabstop>lineEdit_plaka</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_kaydet1</tabstop>
  <tabstop>pushButton_update1</tabstop>
  <tabstop>pushButton_yeni1</tabstop>
  <tabstop>pushButton_sil1</tabstop>
  <tabstop>lineEdit_aracid</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
